### DISCUSSION TITLE
Not getting email notification of push to private repo

### DISCUSSION THREAD

**Discussion by hayden-t** [community/community#53851] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

 i have access to it & set to watch to "all activity" but no emails... yet i get emails for other things.

**Comment by bbkx226** [comment]
Hi @hayden-t , To solve this problem, you should check your email notification settings in your GitHub account. Make sure that you have enabled email notifications for the "Pushes" event in the notification settings of your repository. If this doesn't work, you can try disabling and then re-enabling email notifications for your account. It's also a good idea to check your email's spam folder to make sure the notifications aren't getting filtered there.

**Comment by multikitty** [comment]
Hi @hayden-t ,
1. Go to https://github.com/settings/emails
2. Set your email as Primary email address.
3. Check Email preferences if "Receive all emails, except those I unsubscribe from." is checked.

Hope this will help you!
